src/Approximations/overapproximate.jl

```diff
@@ -1479,3 +1479,34 @@ function overapproximate(Z::Zonotope{N}, ::Type{<:Zonotope}, r::Union{Integer, R
     end
     return Zonotope(c, Gred)
 end
+
+"""
+    overapproximate(am::AffineMap{N, <:AbstractHyperrectangle{N}},
+                    ::Type{<:Hyperrectangle}) where {N}
+
+Overapproximate the affine map of a hyperrectangular set
+using a hyperrectangle.
+
+### Input
+
+- `S`              -- affine map of a hyperrectangular set
+- `Hyperrectangle` -- type for dispatch
+
+### Output
+
+A hyperrectangle.
+
+### Algorithm
+
+If `c` and `r` denote the center and vector radius of a hyperrectangle `H`
+and `v` the translation vector, a tight hyperrectangular overapproximation of
+`M * H + v` is obtained by transforming `c ↦ M*c+v` and `r ↦ abs.(M) * c`, where
+`abs.(⋅)` denotes the element-wise absolute value operator.
+"""
+function overapproximate(am::AffineMap{N, <:AbstractHyperrectangle{N}},
```

Comment:
Consider generalizing to `AbstractAffineMap`.

Reply:
I think this is a good idea now that we have this interface. I haven't tried, but i bet that this will be more efficient than the fallback for a general LazySet.

Reply:
Yes, and for `AffineMap` it should actually be equivalent.

Reply:
Because the fallback uses 2n support function evaluations, while assuming that the matrix, set and vector are readily available, requires two matrix products and one vector sum.